src/main/kotlin/org/cascadebot/bot/events/ReadyListener.kt

```diff
@@ -23,5 +24,7 @@ class ReadyListener(): ListenerAdapter() {
             rabbitMQManager.channel.queueDeclare("shard-$shardId", true, false, false, mapOf())
             rabbitMQManager.channel.queueBind("shard-$shardId", "amq.topic", "shard.$shardId.*.#")
         }
+
+        Main.commandManager.registerCommandsOnce(event.jda)
```

Comment:
We should probably talk about this, considering we're barely going to have any global commands I suppose it's fine, just feels wrong.